public/assets/sass/template/headers/_header.scss

```diff
@@ -218,10 +218,14 @@
     display: flex;
     align-items: center;
     gap: 50px;
-    margin-left: 85px;
+    margin-left: 0px;
```

Comment:
Have you tested this in smaller screens? We need to make sure that our styling changes dont break the site on mobile browsers.

Reply:
These are not written from scratch rather taken from another button's CSS

Reply:
The mentioned CSS property is for the navbar button, and it seems to be working with on all sizes on my local env.